### PR TITLE
Hotfix barcode matching

### PR DESCRIPTION
**Pull request by JamesGlover**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: lib/labware_barcode.rb [hunk 1]
```diff
@@ -18,18 +18,24 @@ def prefix
...
-  def sbcf
-    @sbcf ||= SBCF::SangerBarcode.from_human(@human)
-  end
-
   def to_s
     @human
   end
 
-  delegate :=~, to: :sbcf
+  def =~(other)
```
Comment (andrewsparkes) [inline lib/labware_barcode.rb:25]: Not following this. Is the regex match operator a method name here?

Comment (JamesGlover) [reply lib/labware_barcode.rb:25]: Yes, pretty much all operators in ruby are method names. Essentially this comes from allowing DN1234K to match 1220001234757. I've always read =~ more as a match operator, than explicitly to do with reg-exp, although given Rubocop seems to follow a similar line of reasoning to you, I should probably change this.

Comment (JamesGlover) [reply lib/labware_barcode.rb:25]: Agh, will probably need to updated the SBCF gem as well, due to the use of SBCF::EmptyBarcode, do you mind if I hold off that one for now?

Comment (andrewsparkes) [reply lib/labware_barcode.rb:25]: Sure. Just finding it hard to understand what the method does. Add some comments to explain basic function?

Comment (andrewsparkes) [reply lib/labware_barcode.rb:25]: And what is @sbcf? sanger barcode formatted?
the different versions of barcodes are pretty confusing

Comment (JamesGlover) [reply lib/labware_barcode.rb:36]: Added a bit of explanation

Comment (JamesGlover) [reply lib/labware_barcode.rb:36]: https://github.com/sanger/sanger_barcode_format

**Review by andrewsparkes — APPROVED** [review]

**Pull request merged**